release-notes.md

```diff
@@ -1,3 +1,11 @@
+# Scorpius-19.11.0
+## update Gamine-Services partner permissions##
```

Comment:
call it game services like you wrote in the script name

Reply:
done